nvfuser/__init__.py

```diff
@@ -325,7 +372,14 @@ def execute(
                 _enable_options=_enable_options,
                 _disable_options=_disable_options,
             )
-            return results
+
+            out_tensors: list[torch.Tensor] = []
+            for out_dtensor in out_dtensors:
+                if out_dtensor.mesh.size == 0:
+                    out_tensors.append(out_dtensor.local)
+                else:
+                    out_tensors.append(DistributedTensor(out_dtensor))
```

Comment:
Alternative: I could wrap `_C._DistributedTensor`s as `torch.distributed.tensor.DTensor`s. DTensor pulls in many dependencies and implements many eager operations that nvFuser doesn't need.